src/auro-calendar.js

```diff
@@ -1,6 +1,8 @@
 import { html } from 'lit';
 import styleCss from "./style-auro-calendar-css";
 
+import 'lodash';
```

Comment:
WIth lodash being added to the repo as a dependency can we document why and how this is being used?